student.rb

```diff
@@ -2,13 +2,14 @@
 require_relative './classroom'
 
 class Student < Person
-  def initialize(classroom)
-    super()
-    @classroom = classroom
-  end
-
   # Belong-to relationship
   attr_reader :classroom
+  attr_accessor :parent_permission
+
+  def initialize(age, name, classroom = 'none', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Please fix this error generated  when a user wants to create a new student
![stude](https://user-images.githubusercontent.com/55596970/188282387-91b21a1e-edc8-4af1-bc34-184594235692.png)